packages/story-editor/src/components/panels/document/taxonomies/HierarchicalTermSelector.js

```diff
@@ -61,6 +60,11 @@ const AddNewCategoryArea = styled.div`
   margin: 24px 0 16px;
 `;
 
+const ButtonContainer = styled.div`
+  display: flex;
+  column-gap: 8px;
```

Comment:
I think just `gap` is needed here with flex.

```suggestion
  gap: 8px;
```